site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,416 @@
+
+### Allow MemoryRef to hold a past-the-end offset ([#6141](https://github.com/citra-emu/citra/pull/6141)) by [vitor-k](https://github.com/vitor-k)
+
+Previously, Citra did not correctly understand any copies or operations being held at the end of the emulated VRAM, which would cause them to get completely ignored. This would cause numerous errors in the emulation of games, including some graphics being cut out entirely on HarmoKnight. By allowing MemoryRef, a wrapper over a pointer to physical memory used for serialization, to correctly understand these values, they will no longer be ignored and can be executed correctly.
+
+{{< sidebyside "image" ""
+    "harmoOld.png=What's happened there?"
+    "harmoNew.png=Ah, that's better!" >}}
+
+### arm/dynarmic: More accurate cycle counting ([#6194](https://github.com/citra-emu/citra/pull/6194)) by [merryhime](https://github.com/merryhime)
+
+While most games easily run at full speed nowadays, a long standing issue in Citra has been pre-rendered cutscene videos running way too slowly. This has always been a bit of a mystery, since decoding a video at a resolution of 240p shouldn’t be too taxing on the system of an average user.
+
+By implementing more accurate cycle counting, we are happy to announce we have made a first step towards solving this problem, improving FMV video performance by up to 3x! To understand what this means, we need to take a closer look at how a processor works.
+
+On the lowest level, all that a CPU does is process a list of instructions. Some instructions however take longer to execute than others. The time it takes for an instruction to execute is measured in cycles. Previously, we were using a very naive implementation of assuming every instruction would take one cycle. This leads to games idling longer than they should, therefore unnecessarily slowing down the emulation.
+Now [merryhime](https://github.com/merryhime), the author of [dynarmic](https://github.com/merryhime/dynarmic), has implemented much more accurate cycle counts, significantly reducing this issue. Performance is still problematic due to currently unknown reasons, so please expect FMV videos to run at half speed on our minimum recommended hardware.
+
+## Multiplayer Fixes
+
+### APT: implement Set and GetWirelessRebootInfo ([#5328](https://github.com/citra-emu/citra/pull/5328)) by [B3n30](https://github.com/B3n30)
+
+This PR implements two key services relating to Download Play multiplayer on Citra. `SetWirelessRebootInfo` and `GetWirelessRebootInfo` are used to transfer information and arguments between reboots of Download Play. These are necessary so that the app is able to start another app, like in the way Download Play works via transferring a game demo from one console to another. Download Play does not work in Citra just yet. This PR is just to set up the groundwork for that to hopefully work in the future!
```

Comment:
Might be a good idea to specify the Download Play Applet functionality, rather than just "Download Play". Since users may correlate that as saying that Download Play for local multiplayer doesn't work either, which does work.